src/lib/components/modal/bottomModal/index.tsx

```diff
@@ -70,7 +70,7 @@ export default ({
             </button>
           )}
         </div>
-        <div className={styles.content}>{children}</div>
+        <div className={`${styles.content} pb48`}>{children}</div>
```

Comment:
1. Devs already set the padding of the modal content, this will just increase the bottom spacing on all modals so that content doesn't overlap with intercom, that's why we're adding that to all modals (screenshot below). wdyt?
2. I'll change it 👍  because BottomModal is mostly used on mobile I thought that wasn't needed but it does


![IMG_2893](https://user-images.githubusercontent.com/4015038/231387990-af2d0e25-ac6f-425b-82a1-8a2a25646188.png)

Reply:
Sounds good, thanks for explaining it!